### DISCUSSION TITLE
[3.9.23] com_media does not allow opening folders with an accent

### DISCUSSION THREAD

**Discussion by fredo-m** [joomla/joomla-cms#31650] [2. Cold Features]
Hi,

File managers, like JCE's, allow you to create folders with a French accent in its name.

If the folder is created with an accent, com_media will not open the folder. Clicking on the link returns to the root of the folders.

Regards.

- Joomla! 3.9.23 Stable
- PHP 7.3 FPM

**Comment by coolcat-creations** [comment]
Characters allowed in URLs are limited, I am not sure if its a good practise to allow creating such folders in the media manager. I personally think its wrong and that any other media manager should not allow that. I might be wrong so lets see other opinions.

**Comment by HLeithner** [comment]
Actually a URL is not limited but filesystem's are limited and some utf8 features could lead to security problems.

I personally would not create a file for download using utf8 characters on the other side I use utf8 set URLs.

I convert this to a discussion.
